redwood-lazylayout-uiview/src/commonMain/kotlin/app/cash/redwood/lazylayout/uiview/UIViewLazyList.kt

```diff
@@ -95,6 +98,9 @@ internal open class UIViewLazyList(
           forIndexPath = cellForItemAtIndexPath,
         ) as LazyListContainerCell
 
+        cell.collectionView = collectionView
```

Comment:
I'm not sure whether a cell should hold a reference to the collection view, could it cause a reference cycle? Maybe the var should be a weak reference

Reply:
Or just pass the collection view's frame if that's all we need within the cell

Reply:
Good catch! Just passing in the `CollectionView` frame now.